runtime/src/accounts_db.rs

```diff
@@ -1088,7 +1094,8 @@ pub struct AccountsDb {
     /// GeyserPlugin accounts update notifier
     accounts_update_notifier: Option<AccountsUpdateNotifier>,
 
-    filler_account_count: usize,
+    filler_accounts_count: usize,
+    filler_accounts_size: usize,
```

Comment:
Please replace with single field for `FillerAccountsConfig`, named `filler_accounts_config`.